decidim-core/app/views/decidim/account/_password_fields.html.erb

```diff
@@ -3,11 +3,11 @@
   data-hide-password="<%= t "hide_password", scope: "decidim.devise.shared.password_fields" %>"
   data-hidden-password="<%= t "hidden_password", scope: "decidim.devise.shared.password_fields" %>"
   data-shown-password="<%= t "shown_password", scope: "decidim.devise.shared.password_fields" %>">
-  <%= form.send(:custom_label, :password, t("activemodel.attributes.user.password"), { required: true }) %>
+  <%= form.send(:custom_label, :password, t("activemodel.attributes.user.password"), {}) %>
 
   <% if local_assigns.has_key?(:show_help_text) || local_assigns.has_key?(:user) %>
     <%= form.password_field :password, password_field_options_for(user) %>
   <% else %>
-    <%= form.password_field :password, autocomplete: "current-password", required: true, label: false %>
+    <%= form.password_field :password, autocomplete: "current-password", required: true, label: false, placeholder: t("placeholder_password", scope: "decidim.devise.shared") %>
```

Comment:
On this case we should not add this as a string to translate, as translators (people in Crowdin) will not know what to translate ("*********" is the same in all the languages)
```suggestion
    <%= form.password_field :password, autocomplete: "current-password", required: true, label: false, placeholder: "************" %>
```